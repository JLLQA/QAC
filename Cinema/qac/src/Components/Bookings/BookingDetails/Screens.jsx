import { Container } from 'reactstrap';
import Navbar from '../../Multipage/Navbar/Navbar';
import Deluxe from './Screens/Deluxe_Screen.png';
import Standard from './Screens/Standard_Screen.png';

const Screens = () => {
    return (
        <div>
            <div>
                <Navbar />
            </div>
            <div id="dropped-box" className="container-fluid">
                <Container>
                    <div align="center">
                        <h1>Screens 1 and 2</h1>
                        <h3>Standard Screen</h3>
                        <p>These screens feature </p>
                        <img src='https://c.stocksy.com/a/sS2400/z9/962790.jpg' alt="basicseat"></img>
                        <img id="plan" src={Standard} alt="basicplan"></img>
                        <h1>Screen 3</h1>
                        <h3>Deluxe Screen</h3>
                        <p>We like to refer to this as our "Directors Box" screen.</p>
                        <img src='https://www.showcasecinemas.com/Media/1739/woburnseatsmob2.jpg' alt="deluxeseat"></img>
                        <img id="plan" src={Deluxe} alt="deluxeplan"></img>
                    </div>
                </Container>
            </div>
        </div>
    );
}

export default Screens;