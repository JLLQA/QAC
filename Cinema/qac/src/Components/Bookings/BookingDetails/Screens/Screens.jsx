import { Container } from 'reactstrap';
import Deluxe from './Pictures/Deluxe_Screen.png';
import Standard from './Pictures/Standard_Screen.png';

const Screens = () => {
    document.title = "Screen Information"

    return (
        <div id="dropped-box" className="container-fluid">
            <Container>
                <div align="center">
                    <h1>Screens 1 and 2</h1>
                    <h3>Standard Screen</h3>
                    <p>These screens feature </p>
                    <img src='https://c.stocksy.com/a/sS2400/z9/962790.jpg' alt="basicseat"></img>
                    <img id="plan" src={Standard} alt="basicplan"></img>
                    <h1>Screen 3</h1>
                    <h3>Deluxe Screen</h3>
                    <p>We like to refer to this as our "Directors Box" screen.
                            <br />
                            These VIP seats have extra room to stretch out, a wider chair and are in a prime position,
                            giving you a perfect, unobstructed view of the screen every time.
                        </p>
                    <img src='https://www.showcasecinemas.com/Media/1739/woburnseatsmob2.jpg' alt="deluxeseat"></img>
                    <img id="plan" src={Deluxe} alt="deluxeplan"></img>
                </div>
            </Container>
        </div>
    );
}

export default Screens;