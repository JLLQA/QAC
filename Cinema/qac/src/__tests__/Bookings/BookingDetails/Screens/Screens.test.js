import '@testing-library/jest-dom';
import '@testing-library/user-event';
import { render, screen } from '@testing-library/react';
import React from 'react';
import renderer from 'react-test-renderer';
import Screens from '../../../../Components/Bookings/BookingDetails/Screens/Screens';

test('Screen text is present', () => {
    render(<Screens />);
    const screenText = screen.getByText("Screens");
    expect(screenText).toBeInTheDocument();
});

test("It matches the snapshot", () => {
    const tree = renderer.create(<Screens />).toJSON();
    expect(tree).toMatchSnapshot();
})