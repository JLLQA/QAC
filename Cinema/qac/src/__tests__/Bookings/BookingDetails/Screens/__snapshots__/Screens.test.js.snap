// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It matches the snapshot 1`] = `
<div
  className="container-fluid"
  id="dropped-box"
>
  <div
    className="container"
  >
    <div
      align="center"
    >
      <div
        className="container"
      >
        <div
          className="container"
        >
          <h1>
            Screens 1 and 2
          </h1>
          <h3>
            Classic Screen
          </h3>
          <p>
            These charming screens feature our iconic classic red padded seats, suitable for people of ages.
          </p>
        </div>
        <div
          className="container"
        >
          <img
            alt="basicseat"
            src="https://c.stocksy.com/a/sS2400/z9/962790.jpg"
          />
          <img
            alt="basicplan"
            id="plan"
            src="Standard_Screen.png"
          />
        </div>
      </div>
      <div
        className="container"
      >
        <div
          className="container"
        >
          <h1>
            Screen 3
          </h1>
          <h3>
            Deluxe Screen
          </h3>
          <p>
            We like to refer to this screen as our "Directors Box".
            <br />
            These VIP seats have extra room to stretch out, a wider chair and are in the prime position, giving you a perfect cinematic experience to view films the way they where meant to be seen.
          </p>
        </div>
        <div
          className="container"
        >
          <img
            alt="deluxeseat"
            src="https://www.showcasecinemas.com/Media/1739/woburnseatsmob2.jpg"
          />
          <img
            alt="deluxeplan"
            id="plan"
            src="Deluxe_Screen.png"
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;
